packages/protocol/contracts/signal/SignalService.sol

```diff
@@ -13,7 +13,7 @@ import "./ISignalService.sol";
 contract SignalService is EssentialContract, ISignalService {
     /// @notice Mapping to store the top blockId.
     /// @dev Slot 1.
-    mapping(uint64 chainId => mapping(bytes32 kind => uint64 blockId)) public topBlockId;
+    mapping(uint64 chainId => mapping(bytes32 kind => uint64 blockId)) private _topBlockId;
```

Comment:
None of our services (UI, BE) using this ? If so i guess, fine.

Reply:
will revert this change.